CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [3.0.0](https://github.com/camptocamp/puppet-systemd/tree/3.0.0) (2020-03-09)
+
+[Full Changelog](https://github.com/camptocamp/puppet-systemd/compare/2.8.0...3.0.0)
+
+**Breaking changes:**
+
+- add option for persistent logging [\#127](https://github.com/camptocamp/puppet-systemd/pull/127) ([djvl](https://github.com/djvl))
```

Comment:
As mentioned in my comment at https://github.com/camptocamp/puppet-systemd/pull/127 , we should fix this before we release this. Otherwise we delete the journal of most of our users

Reply:
I'm currently testing https://github.com/camptocamp/puppet-systemd/pull/143 . maybe we can merge that and afterwards to a minor release